devlib/platform/gem5.py

```diff
@@ -127,6 +129,11 @@ def _start_interaction_gem5(self):
             # telnet connection.
             self.stderr_filename = f
 
+            # Enable power modelling if required
+            if self.power_models_dir:
```

Comment:
Sorry, I am not 100% sure to have understood that correctly. If power modelling is activated using completely different approaches depending on the platform scripts (I didn't know that), we probably don't want this PR at all since it will work for only one platform. Is that correct ? Are we always using `POWER_MODEL_ROOT` ? Is it just the `--power-model` argument that's an issue ?